subprojects/kris-io/src/main/kotlin/ch/difty/kris/KRisIO.kt

```diff
@@ -31,7 +33,7 @@ public object KRisIO {
     @JvmStatic
     @Throws(IOException::class)
     public fun process(reader: Reader): List<RisRecord> = runBlocking(Dispatchers.IO) {
-        val lineFlow = BufferedReader(reader).readLines().asFlow()
+        val lineFlow = BufferedReader(reader).lineSequence().asFlow()
         KRis.process(lineFlow).toList()
```

Comment:
```suggestion
        with(BufferedReader(reader)) {
            val lineFlow = lineSequence().asFlow()
            KRis.process(lineFlow).toList().also {
                close()
            }
        }
```
Investigating your suggestion, I realized that we should also close the BufferedReader (that was wrong before your PR. What do you think about above suggestion?